dotcom-rendering/src/web/components/MostViewedFooterLayout.tsx

```diff
@@ -64,26 +63,19 @@ interface Props {
 	sectionName?: string;
 	format: ArticleFormat;
 	ajaxUrl: string;
+	switches: Switches;
+	pageIsSensitive: boolean;
+	isDev?: boolean;
 }
 
 export const MostViewedFooterLayout = ({
 	sectionName,
 	format,
+	switches,
+	pageIsSensitive,
+	isDev,
 	ajaxUrl,
```

Comment:
This is a total nitpick, but it might be good to have the props listed in the same order as the types for easier comparison.